modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_topper.scss

```diff
@@ -95,7 +95,7 @@ html#{$cadmin-selector} {
 			}
 
 			.portlet-content-editable {
-				opacity: 0.5;
+				opacity: 1;
```

Comment:
Hi @wanderlast, some notes:

- This is mostly a product/design call as these changes are product wide (not just the portlets mentioned in the LPS). 
- Said that, as we're focused on A11y at this moment, we can move forward with this product-wide change but I'd like to hear @drakonux @pablo-agulla
- Please consider all breakpoints as there are other `opacity:0.5` settings associated to `.lfr-configurator-visibility` class that kick in when browser window witdth shrinks. @pat270 please help to complete the changes

Reply:
@wanderlast I think it's better to remove the opacity styles rather than updating them to 1. You can remove them from the files:
```
liferay-portal/modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_misc.scss

liferay-portal/modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_topper.scss
```

I also see some styles in:

```
liferay-portal/modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/ddm-templates/styles/ddm-template.ftl
```

I'm not sure if we have to manually update these or if they get updated when the theme / global css files get updated.